{{ cookiecutter.app_name }}/tests/test_settings.py

```diff
@@ -0,0 +1,43 @@
+"""
+Validate that our settings functions work, and we can create yaml files
+"""
+import os
+import tempfile
+import unittest
+
+import mock
+import yaml
+
+from {{ cookiecutter.project_name }}.settings import load_fallback, get_var
+
+
+class TestSettings(unittest.TestCase):
+    """Validate that settings work as expected."""
+
+    def test_load_fallback(self):
+        """Verify our YAML load works as expected."""
+        config_settings = {'TEST_KEY': 'yessir'}
+        _, temp_config_path = tempfile.mkstemp()
```

Comment:
This leaves an open filehandle. Since `os` is already imported, it would be nice to close it.

```python
handle, temp_config_path = tempfile.mkstemp()
os.close(handle)
```